### DISCUSSION TITLE
Integrazione AWOX

### DISCUSSION THREAD

**Discussion by jchrist21** [indomus/forum#1900] [HUB: Home Assistant]
Ciao a tutti,
ho in casa tre plafoniere AWOX correttamente configurate sia in Alexa che tramite app. Funzionano molto bene e vorrei poterle inserire in HA. Dopo aver scaricato AWOX mesh control da Hacs, in impostazioni, dispositivi e servizi, provo ad aggiungere l'integrazione di awox, ma mi restituisce questo errore, dopo un caricamento di qualche minuto.

Errore
Impossibile caricare il flusso di configurazione: undefined

Come posso fare? :(

**Comment by Scialla** [comment]
Dovresti chiederlo agli sviluppatori dell'integrazione..
Consiglio: cerca su google con "awox" + "homeassistant" + l'errore (in inglese)

Vedo che ci sono diverse issue su github
